# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libphonenumber/hotfix-boost-system.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cpp/CMakeLists.txt.vanilla	2026-02-08 13:05:55.513199694 +0100
+++ cpp/CMakeLists.txt	2026-02-08 13:06:00.514412572 +0100
@@ -145,7 +145,7 @@
   if (WIN32)
     set (Boost_USE_STATIC_LIBS ON)
   endif ()
-  find_package (Boost 1.40.0 COMPONENTS date_time system thread)
+  find_package (Boost 1.40.0 COMPONENTS date_time thread)
   if (NOT Boost_FOUND)
     print_error ("Boost Date_Time/System/Thread" "Boost")
   endif ()
